filters/regional/filter_41_POL_CERTPolskaListOfMaliciousDomains/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "filterId": "POL_cert_polska_list_of_malicious_domains",
+    "id": 41,
+    "name": "POL: CERT Polska List of malicious domains",
+    "description": "CERT Polska deals with incidents and threats originating or aiming at systems in the .pl domain or connected to NASK's or other Polish Internet provider's network.",
+    "timeAdded": 1678201318112,
+    "homepage": "https://cert.pl/posts/2020/03/ostrzezenia_phishing/",
```

Comment:
or maybe better `https://incydent.cert.pl/`?

Reply:
Current is better.